### PR TITLE
Update dependencies

### LINKED ISSUE #320: test failing with ember@canary in local development
I see tons of errors like this only with ember@canary.

```
not ok 512 PhantomJS 2.1 - Integration mode | Property | property: looks for elements outside the testing container
    ---
        actual: >
            null
        stack: >
            setupContext@http://localhost:7357/assets/test-support.js:8124:39
            wrapper@http://localhost:7357/assets/test-support.js:8971:39
            http://localhost:7357/assets/test-support.js:7698:30
            initializePromise@http://localhost:7357/assets/vendor.js:63434:15
            Promise@http://localhost:7357/assets/vendor.js:63900:50
            nextStep@http://localhost:7357/assets/test-support.js:7697:52
            tryCatch@http://localhost:7357/assets/vendor.js:63392:22
            invokeCallback@http://localhost:7357/assets/vendor.js:63405:23
            http://localhost:7357/assets/vendor.js:63473:30
            http://localhost:7357/assets/vendor.js:54314:17
            invokeWithOnError@http://localhost:7357/assets/vendor.js:19907:32
            flush@http://localhost:7357/assets/vendor.js:19778:27
            flush@http://localhost:7357/assets/vendor.js:19969:36
            end@http://localhost:7357/assets/vendor.js:20077:47
            _boundAutorunEnd@http://localhost:7357/assets/vendor.js:20039:26
        message: >
            Promise rejected before looks for elements outside the testing container: undefined is not a constructor (evaluating 'this.container.lookupFactory('-view-registry:main')')
```

travis-ci doesn't fail which makes the error more weird. See https://travis-ci.org/san650/ember-cli-page-object/builds/249460658

### PR DESCRIPTION
**Pull request by san650**
Update all dependencies marked as red or yellow in david-dm.org

This PR also removes the blueprint introduced in #274 which is hard to maintain. It's better suited to be its own ember-addon.

Also fixes #320

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -1,47 +1,45 @@
 /*jshint node:true*/
 module.exports = {
+  useYarn: true,
   scenarios: [
     {
-      name: 'ember-1.12',
+      name: 'ember-lts-2.4',
```
Comment (ro0gr) [inline config/ember-try.js]: Why to drop 1.12. I think we definitely support it

Comment (san650) [reply config/ember-try.js]: I'm almost sure the addon still can be installed with ember-1.12, but its really hard to make our addons tests compatible with those old versions.

ember 1.12 is more than two years old which is a long time in js terms, I think we did a good job keeping it for so long, but it's time to move on 😃 .

Unless someone feels too strong about it, I think its best to start doing some changes in the code to make it easier to maintain. That's why I'm removing a whole blueprint in this same PR.

Comment (ro0gr) [reply config/ember-try.js]: Yeah this blueprint looked massive. I hope nobody actively use a `component-test` blueprint 😆

Comment (san650) [reply config/ember-try.js]: Totally, it's a candidate to be its own addon.

https://github.com/san650/ember-cli-page-object/issues/319#issuecomment-345362150

**Pull request merged**